src/gui/properties/proplistdelegate.cpp

```diff
@@ -78,9 +81,10 @@ QWidget *PropListDelegate::createEditor(QWidget *parent, const QStyleOptionViewI
             return nullptr;
     }
 
-    const int priority = index.data(TorrentContentModel::UnderlyingDataRole).toInt();
-    if (static_cast<BitTorrent::DownloadPriority>(priority) == BitTorrent::DownloadPriority::Mixed)
-        return nullptr;
+    // No combobox editor for the "mixed" priority
+    //const int priority = index.data(TorrentContentModel::UnderlyingDataRole).toInt();
+    //if (static_cast<BitTorrent::DownloadPriority>(priority) == BitTorrent::DownloadPriority::Mixed)
+    //    return nullptr;
```

Comment:
You should just remove these instead of commenting them out, squash commits into one and then it will LGTM.

Reply:
Removed & squashed as advised. Thank you.